huawei-wmi.c

```diff
@@ -86,6 +86,8 @@ static const struct key_entry huawei_wmi_keymap[] = {
 	{ KE_IGNORE, 0x293, { KEY_KBDILLUMTOGGLE } },
 	{ KE_IGNORE, 0x294, { KEY_KBDILLUMUP } },
 	{ KE_IGNORE, 0x295, { KEY_KBDILLUMUP } },
+	// Ignore Ambient Light Sensoring
+	{ KE_KEY,		 0x2c1, { KEY_RESERVED } },
```

Comment:
Spacing should match previous ones. Sorry, I'm not trying to be picky, I'm planning on submitting this to kernel upstream and they're very picky 😄

Reply:
Fixed! Don't worry. I should have noticed :)